### DISCUSSION TITLE
It's been more than 2 months but I haven't got access to the new copilot x

### DISCUSSION THREAD

**Discussion by Augustus-PM-Jacobin** [community/community#56200] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Of course, although this situation makes me sad, my question to those who have access is how is copilot x and is it as good as advertised?

**Comment by Akash1134** [comment, marked as answer]
Hi @Augustus-PM-Jacobin ,

While we are unable to guarantee access to Copilot X features, I can add that we have speed up process of admitting users. However, there is no ETA for the access yet. 

Once you get access to preview a feature, please keep in mind that what you are getting access to is considered beta or a technical preview, meaning the feature may still have some kinks to work out.

**Reply by fahim9778** [reply]
@Akash1134 
Same case for me. I have signed up for the waitlist more than 2 months. Still no news. I was super excited and still am, but it feels frustrating at moments when people around me (like enterprise users) already got the access but not poor soul like me. I have using Copilot since last year and would absolutely love the new chat feature 😭

**Comment by davecheney** [comment]
We have onboarded 60% of those on the waitlist, please be patient, it won't be long now.

nb. Currently Copilot for Business seats are not eligible for Copilot Chat.

**Reply by WillooWisp** [reply]
@davecheney I just converted my Individual Copilot seat to a Business seat, and now I read this... I am on the waitlist since a month ago, but is your recommendation to convert back to an individual account then since business is not supported?

**Reply by davecheney** [reply]
Please have your admin talk to their sales representative. I believe the CfB restriction may be lifted shortly.

**Reply by fahim9778** [reply]
> We have onboarded 60% of those on the waitlist, please be patient, it won't be long now.
> 
> nb. Currently Copilot for Business seats are not eligible for Copilot Chat.

hello @davecheney 
I just got the access to Copilot Chat private beta. Thanks!
